.github/workflows/linux.yml

```diff
@@ -358,34 +358,3 @@ jobs:
           name: dosbox-staging-linux-x86_64
           path: dosbox-staging-linux-${{ env.VERSION }}.tar.xz
           overwrite: true
-
-
-  # This job exists only to publish an artifact with version info when building
-  # from main branch, so snapshot build version will be visible on:
-  # https://dosbox-staging.github.io/downloads/devel/
-  #
-  publish_additional_artifacts:
-    name: Publish additional artifacts
```

Comment:
So this whole CI mess was just to generate a version string for the website?  Good riddance.

Reply:
Yeah, plus to generate a changelog that probably no-one ever looked at. I certainly did not know it even existed until I started looking into these workflows more deeply 😆